solana-programs/anchor/audius-data/programs/audius-data/src/lib.rs

```diff
@@ -865,9 +894,15 @@ pub struct RemoveUserAuthorityDelegate<'info> {
         seeds = [&user.key().to_bytes()[..32], &user_authority_delegate.to_bytes()[..32]],
         bump = delegate_bump
     )]
-    pub user_authority_delegate_pda: Account<'info, UserAuthorityDelegate>,
+    pub new_user_authority_delegate: Account<'info, UserAuthorityDelegate>,
```

Comment:
this should be renamed from `new_user_authority_delegate` to `current_user_authority_delegate` probably